### ISSUE TITLE
Will n-D observation states be supported? 

### ISSUE THREAD

**Issue by GreatArcStudios** [facebookresearch/Pearl#39]
From the util function (and testing with `ALE/Bowling-v5`), it appears that only 1-D (vector) observation shapes are supported. Allowing for n-D shapes can be interesting as it can open up experiments things like text-embeddings, while maintaining the given structure. 

Is there a plan to add this in?

**Comment by rodrigodesalvobraz** [comment]
Yes, we plan to implement such features in time. I am adding this to the TODO list. Thank you!
